pipelines/vars/foreman_base.yml

```diff
@@ -0,0 +1,12 @@
+foreman_installer_scenario: foreman
+foreman_installer_options_internal_use_only:
+  - "--foreman-initial-admin-password {{ foreman_installer_admin_password }}"
+  - "--puppet-server-max-active-instances 1"
+  - "--puppet-server-jvm-min-heap-size 1G"
+  - "--puppet-server-jvm-max-heap-size 1G"
+server_box:
+  box: "{{ foreman_os }}"
```

Comment:
Another thought: this could also use `{{ foreman_os }}-foreman-{{ foreman_version }}`. Then it would inherit all the defaults from [versions.yaml](https://github.com/theforeman/forklift/blob/master/vagrant/config/versions.yaml). This is important when we're going to switch the Puppet version (which I'm about to do). It also means you don't have to set the `foreman_repositories_version` and related katello/pulp versions.

Reply:
I like that!

Reply:
So this does not work as intended right now, as we only set the right `*_version` vars inside the ansible provisioner, which we don't use during pipelines. I'll see what we can do here.